modules/osdk-updating-128-to-131.adoc

```diff
@@ -40,39 +40,41 @@ The following procedure updates an existing {type}-based Operator project for co
 
 .Procedure
 
-ifdef::golang,hybrid,java[]
-* Edit your Operator project's makefile to update the Operator SDK version to {osdk_ver}, as shown in the following example:
+. Edit your Operator project's Makefile to update the Operator SDK version to `v{osdk_ver}-ocp`, as shown in the following example:
 +
-.Example makefile
+.Example Makefile
 [source,make,subs="attributes+"]
 ----
 # Set the Operator SDK version to use. By default, what is installed on the system is used.
 # This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
-OPERATOR_SDK_VERSION ?= v{osdk_ver} <1>
+OPERATOR_SDK_VERSION ?= v{osdk_ver}-ocp
 ----
-<1> Change the version from `{osdk_ver_n1}` to `{osdk_ver}`.
-endif::[]
 
-ifdef::helm[]
-. Edit your Operator's Dockerfile to update the Helm Operator version to {osdk_ver}, as shown in the following example:
+. Find the `ose-kube-rbac-proxy` pull spec in the following files, and update the image tag to `v{product-version}`:
```

Comment:
In the Preview docs `v{product-version}`  is "vBranch Build". Is this expected?

Reply:
Yes, `{product-version}` will render as `4.17` on the `enterprise-4.17` branch when it is merged and published. It will show as `Branch Build` only during PR preview stage.